nodes/text/mn_split_text.py

```diff
@@ -42,11 +45,16 @@ def getOutputSocketNames(self):
 
 	def execute(self, text, splitBy):
 		textList = []
+
 		if self.splitType == "Characters": textList = list(text)
 		elif self.splitType == "Words": textList = text.split()
 		elif self.splitType == "Lines": textList = text.split("\n")
-		elif self.splitType == "Custom":
-			if splitBy == "": textList = list(text)
-			else: textList = text.split(splitBy)
+
+		elif self.splitType == "Regexp":
+			if splitBy == "": textList = [text]
+			else: 
+				if self.keepDelimiters == True: textList = re.split("("+splitBy+")", text)
```

Comment:
Yeah that's true. I can remove it if you want. Personally I don't mind and sometimes find it clearer to read :)